examples/app.js

```diff
@@ -8,8 +8,13 @@ function configureLogging() {
             .map(message => {
                 if (message instanceof Error) {
                     const { stack, ...rest } = message;
+                    if (!rest) {
+                        console.debug('aaaa', message);
+                        return stack;
+                    }
                     return `${JSON.stringify(rest, null, 2)}\n${stack}`;
                 } else if (typeof message === 'object') {
+                    console.debug('aaaaaaa', message);
```

Comment:
Do we want to remove these console messages?

Reply:
+1

Reply:
Great catch! Accidentally left those there